pkg/analytics/analytics.go

```diff
@@ -247,6 +247,8 @@ func (a *AnalyticsImpl) GetOutputData() *analyticsOutput {
 	return output
 }
 
+var userCurrent = user.Current
+
```

Comment:
I'd feel better about this if it were an argument to `New`, or a setter like `SetClient` (depending on the actual use case for those functions; don't wanna mix them)

Reply:
Good suggestion, updated!